spec/lib/tasks/migrate_pacbio_aliquot_data.rake_spec.rb

```diff
@@ -127,6 +127,12 @@
       wells = create_list(:pacbio_well, 5, pool_count: 2)
       pools = wells.map(&:pools).flatten
 
+      # Get rid of aliquots that were created by the factory
+      pools.each do |pool|
+        pool.primary_aliquot.destroy
+        pool.used_aliquots.destroy_all
+      end
```

Comment:
The gem we use to create dummy data is called factory bot and you generate data by using factories. The factories in this case are already updated to support aliquots but since this rake task test wants data without aliquots we get rid of any aliquots that were previously created.

Reply:
Ok, now I got it. Thank you for explaining the details.